shared/src/business/entities/User.ts

```diff
@@ -202,6 +202,10 @@ export class User extends JoiValidationEntity {
   getValidationRules() {
     return User.VALIDATION_RULES;
   }
+
+  public setisUpdatingInformation(value: boolean) {
```

Comment:
Is this method used anywhere?

Reply:
shared/src/business/useCases/users/updateUserContactInformationInteractor.ts
line 108